concepts/teams-changenotifications-teammembership.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Get change notifications for any change in Teams membership using Microsoft Graph"
+description: "Get change notifications for any change in Teams membership using Microsoft Graph"
+author: "Abhishek Anand"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+ms.custom: scenarios:getting-started
+---
+
+# Get change notifications for change in teams membership using Microsoft Graph
+
+Change notifications enable you to subscribe to changes (create, update, and delete) to [membership](/graph/api/resources/chatMessage?preserve-view=true) in a [team](/graph/api/resources/team?preserve-view=true). Change notifications provide a low latency model by allowing you to maintain a [subscription](/graph/api/resources/webhooks?preserve-view=true). You can also get the resource data in the notifications and therefore avoid calling the API to get the payload.
```

Comment:
'membership' hyperlink is wrong. I think we didn't replace all instances of 'chat message' in this doc yet

Reply:
and for all 3 pages, we haven't mentioned the properties which won't be part of the models